components/automate-cli/cmd/chef-automate/restart.go

```diff
@@ -49,6 +109,199 @@ func runRestartServices(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func init() {
-	RootCmd.AddCommand(restartServicesCmd)
+func runRestartFromBastion(flags *RestartCmdFlags, rs RemoteCmdExecutor, nu NodeOpUtils, printRestartOutput func(map[string][]*CmdResult, string, *cli.Writer)) error {
+	infra, _, err := nu.getHaInfraDetails()
+	if err != nil {
+		return err
+	}
+	if flags.timeout < DEFAULT_TIMEOUT_FOR_RESTART {
+		return errors.Errorf("The operation timeout duration for each individual node during the restart should be set to a value greater than %v seconds.", DEFAULT_TIMEOUT_FOR_RESTART)
+	}
+	if !flags.automate && !flags.chefServer && !flags.opensearch && !flags.postgresql {
+		if len(flags.node) != 0 {
+			return status.Errorf(status.InvalidCommandArgsError, "Please provide service flag")
+		}
+		flags.automate = true
+		flags.chefServer = true
+		if !isManagedServicesOn() {
```

Comment:
Don't call function directly use nodeUtils to call

Reply:
done